### ISSUE TITLE
[Feature Request] Optimize the position of call icon

### ISSUE THREAD

**Issue by ChenfaZheng** [telegramdesktop/tdesktop#24111]
### Is your feature request related to a problem?

It's quite often to touch this call icon by mistake when I actually want to minimize the window. It immediately calls to my friend and could be embarrassing and annoying.
<img width="402" alt="call button" src="https://user-images.githubusercontent.com/80394200/155834987-cef397a0-db96-4029-b97d-a88a75463fee.png">

### Describe the solution you'd like

Move it next to the user info (just away from the top-right corner), or provide an option to hide it.

### Describe alternatives you've considered

Basically it doesn't affect to current functions.

**Comment by Aokromes** [comment]
now adicional click is needed to make call, it's ok that for you?

**Comment by ChenfaZheng** [comment]
that's great 👍
